src/rt/minfo.d

```diff
@@ -368,7 +368,8 @@ struct ModuleGroup
 
                                     string errmsg = "";
                                     buildCycleMessage(idx, midx, (string x) {errmsg ~= x;});
-                                    throw new Error(errmsg, __FILE__, __LINE__);
+                                    assert(0, errmsg);
```

Comment:
Is this right though ? That's an user-facing message, not a programming error

Reply:
The assert gives a user-facing message. Besides, being unable to run the constructors is a programming error.